app/controllers/api/user_programs_controller.rb

```diff
@@ -1,12 +1,15 @@
 class Api::UserProgramsController < ApplicationController
   before_action :authenticate_user!
 
-
   def index(page: nil)
-    @programs = current_user.programs.unchecked
-                  .where('started_at < ?', Date.tomorrow + 1.day + 5.hours)
-                  .includes(:channel, :work, episode: [:work])
-                  .order(started_at: :desc)
-                  .page(page)
+    @programs = current_user.
+                  programs.
+                  unchecked.
+                  work_published.
+                  episode_published.
+                  where('started_at < ?', Date.tomorrow + 1.day + 5.hours).
```

Comment:
Align the operands of an expression in an assignment spanning multiple lines.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.